### PR TITLE
audtool: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 4.3.1

### CODE REVIEW

File: pages/common/audtool.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# audtool
+
+> Control Audacious using commands.
+> More information: <https://manned.org/audtool>.
+
+- Play/Pause audio playback:
+
+`audtool playback-playpause`
```
Comment (spageektti) [inline pages/common/audtool.md:8]:
```suggestion
`audtool --playback-playpause`
```

File: pages/common/audtool.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Print artist, album, and song name of currently playing song:
+
+`audtool current-song`
```
Comment (spageektti) [inline pages/common/audtool.md:12]:
```suggestion
`audtool --current-song`
```

File: pages/common/audtool.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Set volume of audio playback:
+
+`audtool set-volume {{100}}`
```
Comment (spageektti) [inline pages/common/audtool.md:16]:
```suggestion
`audtool --set-volume {{100}}`
```

File: pages/common/audtool.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Skip to the next song:
+
+`audtool playlist-advance`
```
Comment (spageektti) [inline pages/common/audtool.md:20]:
```suggestion
`audtool --playlist-advance`
```

File: pages/common/audtool.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Print the bitrate of the current song in kilobits:
+
+`audtool current-song-bitrate-kbps`
```
Comment (spageektti) [inline pages/common/audtool.md:24]:
```suggestion
`audtool --current-song-bitrate-kbps`
```

File: pages/common/audtool.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- Open audacious in full-screen if hidden:
+
+`audtool mainwin-show`
```
Comment (spageektti) [inline pages/common/audtool.md:28]:
```suggestion
`audtool --mainwin-show`
```

File: pages/common/audtool.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+- Display help:
+
+`audtool help`
```
Comment (spageektti) [inline pages/common/audtool.md:32]:
```suggestion
`audtool --help`
```

File: pages/common/audtool.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+- Display settings:
+
+`audtool preferences-show`
```
Comment (spageektti) [inline pages/common/audtool.md:36]:
```suggestion
`audtool --preferences-show`
```

File: pages/common/audtool.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
+# audtool
+
+> Control Audacious using commands.
+> More information: <https://manned.org/audtool>.
+
+- Play/Pause audio playback:
```
Comment (CleanMachine1) [inline pages/common/audtool.md:6]:
```suggestion
- Play/pause audio playback:
```

**Review by spageektti — APPROVED** [review]
LGTM, thanks for your contribution. In the manual all the options are preceded by `--` so I think it should be so here too.

**Comment by vitorhcl** [comment]
> LGTM, thanks for your contribution. In the manual all the options are preceded by `--` so I think it should be so here too. 
> 
> edit: I downloaded the tool to check and the help says `Commands may be prefixed with '--' (GNU-style long options) or not, your choice.` but my suggestion still should be considered.

IMO we could keep them without "--" because they work like sub-commands, not like options that can be combined, i.e., they are mutually exclusive.

This would make the exclusivity clear just by glancing at the examples, even without reading the descriptions.

**Pull request merged**